### PR TITLE
Adds dialog for completed workout in Fitness demo

### PR DESCRIPTION
**Pull request by vlidholt**

### CODE REVIEW

File: examples/material_gallery/lib/demo/fitness_demo.dart [hunk 1]
```diff
@@ -489,3 +509,70 @@ class _JumpingJackPart extends Sprite {
...
+
+  _FireworksNode fireworks;
+
+  Widget build(BuildContext context) {
+    return new SpriteWidget(fireworks);
+  }
+}
+
+class _FireworksNode extends NodeWithSize {
+  _FireworksNode() : super(const Size(1024.0, 1024.0));
```
Comment (eseidelGoogle) [inline examples/material_gallery/lib/demo/fitness_demo.dart]: Should the magical constant 1024 be factored out somewhere?

Comment (vlidholt) [inline examples/material_gallery/lib/demo/fitness_demo.dart]: Can do!

File: examples/material_gallery/lib/demo/fitness_demo.dart [hunk 2]
```diff
@@ -8,6 +8,7 @@ import 'dart:ui' as ui;
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_sprites/flutter_sprites.dart';
+import 'package:vector_math/vector_math_64.dart' as vec;
```
Comment (abarth) [inline examples/material_gallery/lib/demo/fitness_demo.dart]: We usually import this without qualification.

**Comment by abarth** [comment]
lgtm

**Pull request merged**